src/eko/evolution_operator/__init__.py

```diff
@@ -446,10 +433,8 @@ def compute(self):
             self.config["method"],
         )
 
-        # TODO: which is the optimal numbers of pool to open?
-        # make it an external parameter?
         # run integration in parallel for each grid point
-        with Pool(8) as pool:
+        with Pool(int(os.cpu_count() / 2)) as pool:
```

Comment:
Keep the default, it's good enough and much simpler

Reply:
Have you tried it?
I tried with the full number and thought was too much..

Reply:
Fine, I was just saying to keep it simple, if it's slightly suboptimal we don't need.

I steel hope this to be needed only temporary.